.circleci/config.yml

```diff
@@ -24,10 +24,7 @@ jobs:
           name: Run tests
           command: |-
               ./build/linux/amd64/earthly --ci -P \
-              +test --DOCKERHUB_MIRROR_AUTH_FROM_CLOUD_SECRETS=true
-      - run:
-          name: Execute fail test
-          command: "! ./build/linux/amd64/earthly --ci +test-fail"
```

Comment:
Should we leave this one in?

Reply:
it's covered under githubactions already:

```
.github/workflows/reusable-test.yml:        run: "! ${{inputs.SUDO}} ${{ inputs.BUILT_EARTHLY_PATH }} ${{inputs.EXTRA_ARGS}} --ci ./tests/fail+test-fail"
```

having it tested in circleci doesn't provide any extra coverage.